JSQMessagesViewController/Views/JSQMessagesComposerTextView.m

```diff
@@ -156,6 +156,13 @@ - (void)setTextAlignment:(NSTextAlignment)textAlignment
     [self setNeedsDisplay];
 }
 
+- (void)paste:(id)sender
+{
+    if (!self.pasteDelegate || [self.pasteDelegate composerTextView:self shouldPasteWithSender:sender]) {
```

Comment:
let's remove `!self.pasteDelegate ||`. if the delegate is `nil`, this will evaluate to false.